tests/settings/e2e/steps/currency-display-settings.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import { When, Then} from 'cucumber';
+
+const CONVERTED_CURRENCY_WALLET_DISPLAY_GENERAL = '//*[@class="WalletSummaryCurrency_currencyRate"]';
+const DISPLAY_BALANCE_OTHER_CURRENCYS_TOGGLE_STATUS_OFF = '//*[@class="SimpleSwitch_switch SwitchOverrides_switch"]';
+const DISPLAY_BALANCE_OTHER_CURRENCYS_TOGGLE_STATUS_ON = '//*[@class="NormalSwitch_component SimpleSwitch_root SwitchOverrides_root SimpleSwitch_checked SwitchOverrides_checked"]';
+const SELECT_CURRENCY_DROPDOWN = '//*[@label="Select currency"]';
+
+When(/^I open currency selection dropdown$/, function() {
+  return this.waitAndClick(SELECT_CURRENCY_DROPDOWN);
+});
+
+When(/^I select "([^"]*)" as the selected displayed currency$/, function(currency) {
```

Comment:
If you like you could use arrow functions and you get smaller code 😉

Reply:
Hi Daniel. I agree it looks a lot better to use arrow functions. I will definetly incorporate them in future.But in this context I got this error 
<img width="1256" alt="Screenshot 2021-07-16 at 10 18 42" src="https://user-images.githubusercontent.com/56027085/125938934-92dec9ee-0d7c-4a95-a543-50a95700cce0.png">
<img width="519" alt="Screenshot 2021-07-16 at 12 09 55" src="https://user-images.githubusercontent.com/56027085/125938942-fb1f2be9-98b8-4b82-bfeb-843b15515145.png"> It seems arrow functions are not suitable here because they are not suitable for binding methods as it changes the scope(or is unsuitable for passing the scope) of what this is ?

Reply:
Arrow functions do not work here because cucumber steps are bound to the shared testing context (called "world" in cucumber terms)